### ISSUE TITLE
Boost 1.72

### ISSUE THREAD

**Issue by JonasMueller1991** [phbasler/BSMPT#21]
Most recent version boost produce following note/warning while compiling:
```shell
/usr/local/include/boost/math/interpolators/cubic_b_spline.hpp:28:1: note: #pragma message: This header is deprecated. Use <boost/math/interpolators/cardinal_cubic_b_spline.hpp> instead.
   28 | BOOST_HEADER_DEPRECATED("<boost/math/interpolators/cardinal_cubic_b_spline.hpp>");
      | ^~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by phbasler** [comment]
@JonasMueller1991 Please test the "boost" branch with a boost version 1.72 or higher and with a boost version < 1.71 if both work and provide the same result in EWBG

**Comment by phbasler** [comment]
Closed by release of 2.1
